autoware_adapi_v1_msgs/vehicle/msg/GeoPoint.msg

```diff
@@ -0,0 +1,2 @@
+float64 latitude
+float64 longitude
```

Comment:
@isamu-takagi @tkhmy What is geo point?

Reply:
@yukkysaito 
Yes, is the global position.
We plan to extract the MGRS Area Code from map and combine with the pose to input into this function, do you think is feasible? 
https://github.com/autowarefoundation/autoware_common/blob/main/tmp/lanelet2_extension/include/lanelet2_extension/projection/mgrs_projector.hpp#L69

Reply:
I see. I think it would be better to have a frame id, what do you think?
@mitsudome-r Can you think of any other fields that would be needed?

Reply:
I see, okay, I will add header which include 'base_link` as the frame id

Reply:
I think it needs a new frame_id for the global position.

Reply:
Maybe `Altitude` can be added.

Reply:
@xmfcx Current we do not able to get the altitude from MGRS, so even if we put it in, we only can get 0

Reply:
We will handle Altitude in Autoware eventually, so it's better to have it in the API.

Reply:
@xmfcx @yukkysaito @isamu-takagi 
About the `frame_id` for geographic_pose, I not able to find any good refer naming for it.
I'm thinking to use `global` as the naming, do you think?